openrr-apps/src/bin/robot_teleop.rs

```diff
@@ -155,7 +155,29 @@ async fn main() -> Result<()> {
         }
         #[cfg(windows)]
         GamepadKind::Builtin(BuiltinGamepad::Keyboard) => {
-            tracing::warn!("`gamepad = \"Keyboard\"` is not supported on windows");
+            anyhow::bail!("`gamepad = \"keyboard\"` is not supported on windows");
+        }
+        #[cfg(feature = "ros")]
+        GamepadKind::Builtin(BuiltinGamepad::JoyGamepad) => {
+            let mut button_map = HashMap::new();
+            button_map.insert(0, arci::gamepad::Button::West);
+            button_map.insert(1, arci::gamepad::Button::South);
+            button_map.insert(2, arci::gamepad::Button::East);
+            button_map.insert(3, arci::gamepad::Button::North);
+            let mut axis_map = HashMap::new();
+            axis_map.insert(0, arci::gamepad::Axis::LeftStickX);
+            axis_map.insert(1, arci::gamepad::Axis::LeftStickY);
+            axis_map.insert(2, arci::gamepad::Axis::RightStickX);
+            axis_map.insert(5, arci::gamepad::Axis::RightStickY);
+            axis_map.insert(6, arci::gamepad::Axis::DPadY);
+            axis_map.insert(7, arci::gamepad::Axis::DPadX);
+            switcher
+                .main(arci_ros::JoyGamepad::new("joy", button_map, axis_map))
+                .await;
```

Comment:
I copied these default setting from arci-ros's example, but I'm not sure what is the correct default here.

Reply:
@kaaatsu32329 Could you try this?
Anyway, ROS joy_node can be configurated from ROS parameter system. It might not be so much problem if the button/axis bind is not comfortable. (But we recommend PS4/PS5 joy in openrr, it should be for PS4 as default)